hardpy/pytest_hardpy/db/storage_interface.py

```diff
@@ -0,0 +1,62 @@
+# GNU General Public License v3.0 (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import annotations
+
+from abc import ABC, abstractmethod
+from typing import TYPE_CHECKING, Any
+
+if TYPE_CHECKING:
+    from pydantic._internal._model_construction import ModelMetaclass
+
+
+class IStorage(ABC):
```

Comment:
nitpick: name should not contain java style "I" prefix. Using just `Storage` would be more pythonic.